shell/app/electron_content_client.cc

```diff
@@ -128,7 +128,8 @@ void ComputeBuiltInPlugins(std::vector<content::PepperPluginInfo>* plugins) {
   // here.
   content::WebPluginInfo info;
   info.type = content::WebPluginInfo::PLUGIN_TYPE_BROWSER_PLUGIN;
-  info.name = base::UTF8ToUTF16("Chromium PDF Viewer");
+  const char16_t kPDFExtensionPluginName[] = u"Chromium PDF Viewer";
+  info.name = kPDFExtensionPluginName;
```

Comment:
Any reason this can't be

```suggestion
  info.name = u"Chromium PDF Viewer";
```

?